tests/includes/juju.sh

```diff
@@ -276,6 +276,10 @@ pre_bootstrap() {
 		export BOOTSTRAP_ADDITIONAL_ARGS="${BOOTSTRAP_ADDITIONAL_ARGS:-} --model-default agent-stream=build-${SHORT_GIT_COMMIT}"
 	fi
 
+	if [[ -n ${BOOTSTRAP_ARCH:-} ]]; then
```

Comment:
are bootstrap-constraints set elsewhere? so we are sure we're not going to conflict